lib/menu.rb

```diff
@@ -0,0 +1,19 @@
+require 'takeaway.rb'
+require 'order.rb'
+class Menu
+  attr_reader :dishes
+  DISHES_LIST = {
+    "Chicken Rice" => 4.50,
+    "Fried Chicken" => 5.50,
+    "Fried Rice" => 2.50,
+    "Side Salad" => 1.50,
+    "Noodle Soup" => 3.50
+  }
```

Comment:
You could improve your code by not hard-coding the menu: https://github.com/nboyse/takeaway-challenge/blob/master/docs/review.md#open-closed-principle